i18n/en.json

```diff
@@ -4180,15 +4180,6 @@
   "sidebar.tutorialProductSwitcher.admin": "System Admins can also configure and customize Mattermost using the **System Console** and install Mattermost Apps from the **Marketplace**.",
   "sidebar.tutorialProductSwitcher.switchProducts": "Launch other Mattermost products from here, including **Boards** and **Playbooks**.",
   "sidebar.tutorialProductSwitcher.title": "Try Boards and Playbooks",
-  "sidebar.tutorialScreen1.body": "**Channels** organize conversations across different topics. They're open to everyone on your team. To send private communications use **Direct Messages** for a single person or **Private Channels** for multiple people.",
-  "sidebar.tutorialScreen1.title": "Channels",
-  "sidebar.tutorialScreen2.body1": "Here are two public channels to start:",
-  "sidebar.tutorialScreen2.body2": "**{townsquare}** is a place for team-wide communication. Everyone in your team is a member of this channel.",
-  "sidebar.tutorialScreen2.body3": "**{offtopic}** is a place for fun and humor outside of work-related channels. You and your team can decide what other channels to create.",
-  "sidebar.tutorialScreen2.title": "\"{townsquare}\" and \"{offtopic}\" channels",
-  "sidebar.tutorialScreen3.body1": "Click **\"More...\"** to create a new channel or join an existing one.",
-  "sidebar.tutorialScreen3.body2": "You can also create a new channel by clicking the **\"+\" symbol** next to the public or private channel header.",
-  "sidebar.tutorialScreen3.title": "Creating and Joining Channels",
```

Comment:
did you meant to remove these? alongside the `sidebar_tip.tsx`?

Reply:
Yeah, as they aren’t getting used anymore but I think we should ask @esethna should we remove this tutorial tip, this isn’t getting used anymore.